examples/repl/main.rs

```diff
@@ -321,7 +321,7 @@ async fn start(args: Vec<String>) -> Result<(), Error> {
                 Ok(line) => {
                     // TODO: ignore "set mail_pw"
                     rl.add_history_entry(line.as_str());
-                    async_std::task::block_on(reader_s.send(line));
+                    async_std::task::block_on(reader_s.send(line)).unwrap();
```

Comment:
This [`SendError`](https://docs.rs/async-std/1.8.0/async_std/channel/struct.SendError.html) shouldn't happen because the channel should never be closed, but I'd rather print the error instead of panic here.

Reply:
It shouldn't happen, and it is just the repl, which is why an `unwrap` seems appropriate to me  here